src/logger/initReporters.ts

```diff
@@ -12,16 +13,23 @@ export function initReporters(config: Config) {
     logLevel = LogLevel[config.logLevel as LogLevelString];
   }
 
-  const reporters = [
+  const reporters: Array<AdoReporter | JsonReporter | NpmLogReporter> = [
```

Comment:
Please add a "reporter" type to this via a string rather than using an environment variable.

Reply:
This is fine - in the future, we should think about how to clean this up. We should allow for multiple reporter types.